### ISSUE TITLE
Reenable legacy DATs for snowflake + bigquery

### ISSUE THREAD

**Issue by edgao** [airbytehq/airbyte#29818]
tl;dr revert https://github.com/airbytehq/airbyte/pull/29783/commits/b244993357ef17c7062cba74fefcc337d1cc6a64 and make DATs pass

https://github.com/airbytehq/airbyte/pull/29783 makes all snowflake/bigquery syncs run in 1s1t mode. It has a few updates to their DAT classes to make some tests pass, but there are still some problems:
* `testIncrementalDedupeSync` fails with `expected: <7> but was: <5>`, presumably because we're now deduping raw tables
* `testCustomDbtTransformationsFailure` fails with `Could not find image: airbyte/normalization-snowflake:dev`, probably because we're not building normalization? ... though there's still a question about why we're using normalization at all here
* `testSyncWithNormalization` fails with `AssertionFailedError: expected:  but was: `, which is not particularly helpful, but probably related to the lack of normalization

**Comment by evantahler** [comment]
Maybe we remove all reference to normalization from DATs

**Comment by cynthiaxyin** [comment]
Updated to include dest Redshift.

For future reference, here is [the list of failing DATs](https://docs.google.com/spreadsheets/d/1HxJ_b0HJgv18R1VK9Y8YHPzB8DAT7BlT_3t-ii8-Djk/edit#gid=0) + [the original CI log from Dagger](https://drive.google.com/file/d/1l1l6uaYx4P1g0jgsuMWZdumPVsBJsPPI/view?usp=sharing).
